module/build.sbt

```diff
@@ -56,11 +51,11 @@ ReleaseKeys.releaseProcess := Seq[ReleaseStep](
   commitReleaseVersion,
   tagRelease,
   ReleaseStep(
-    action = state => Project.extract(state).runTask(PgpKeys.publishSigned, state)._1,
+    action = Command.process("publishSigned", _),
     enableCrossBuild = true
   ),
   setNextVersion,
   commitNextVersion,
-  ReleaseStep(state => Project.extract(state).runTask(SonatypeKeys.sonatypeReleaseAll, state)._1),
+  ReleaseStep(action = Command.process("sonatypeReleaseAll", _)),
```

Comment:
I've recently been through the pain of upgrading sbt-release on a cross-building project that publishes signed artifacts to Sonatype (the perfect storm!). This is what I got to work:

```
releaseCrossBuild := true
releasePublishArtifactsAction := publishSigned.value
releaseProcess := Seq(
  checkSnapshotDependencies,
  inquireVersions,
  runClean,
  runTest,
  setReleaseVersion,
  commitReleaseVersion,
  tagRelease,
  publishArtifacts,
  setNextVersion,
  commitNextVersion,
  releaseStepCommand("sonatypeReleaseAll"),
  pushChanges
)
```